DSCResources/MSFT_xDNSServerAddress/MSFT_xDNSServerAddress.psm1

```diff
@@ -48,15 +48,14 @@ function Get-TargetResource
         $($LocalizedData.GettingDNSServerAddressesMessage)
         ) -join '')
 
+    $dnsClientServerAddresses = Get-NetAdapter -Name $InterfaceAlias | ForEach-Object { Get-DnsClientServerAddress -InterfaceAlias $_  -AddressFamily $AddressFamily }
```

Comment:
Appreciate. Almost all style guidance was what I'm always doing, without 100 char line. I will follow it.